docs/integrations/storage.md

```diff
@@ -215,3 +215,28 @@ which means that the code would work with the object where it's originally store
 
 To enable direct access, specify the urls to access directly.
 
+### Path Substitution
+The ClearML StorageManager supports local path substitution when fetching files.<br/>
```

Comment:
@ainoam it does, but what it does is that it creates a new paragraph and not just a new line (similar to <br/><br/>). I was trying to convey that this is a new line in the same paragraph, and as far as I remember, it's what I used when I originally wrote the docs. Do you think differently?

Reply:
@erezalg Nope, their the same (for "generic" markdown). But we can keep it since there's too much mixed usage anywho - Just keep in mind for future.